packages/mdc-list/foundation.js

```diff
@@ -207,6 +162,16 @@ class MDCListFoundation extends MDCFoundation {
     if (listItemIndex >= 0) {
       this.adapter_.setTabIndexForListItemChildren(listItemIndex, -1);
     }
+
+    /**
+     * Between Focusout & Focusin some browsers do not have focus on any element. Setting a delay to wait till the focus
+     * is moved to next element.
+     */
+    setTimeout(() => {
```

Comment:
I found a bug where checkbox based list end up having multiple tab stops.

Steps to reproduce:

1. Add `tabindex=0` to selected checkbox item.
2. programmatically set selected index. (Shouldn't change the tabindex state)
3. Check other list items. Since there is no trace of list item that had `tabindex=0` during render, it ends up adding second tab stop on last selected / checked list item.

Reply:
Yes, the foundation was already doing that. But earlier it wasn't changing the tabindex on focus change, it remains at selected item which makes it easier to receive focus back at selected item.

This change fixes #3611